pyproject.toml

```diff
@@ -1,3 +1,61 @@
 [build-system]
-requires = ["setuptools >= 56.2.0", "gitpython >= 3.1.18", "wheel"]
-build-backend = "setuptools.build_meta:__legacy__"
\ No newline at end of file
+requires = ["setuptools>=61",  "gitpython >= 3.1.18", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "picca"
+dynamic = ["dependencies", "version", "readme"]
+description = "Package for Igm Cosmological-Correlations Analyses"
+requires-python = ">=3.7"
+license = "GPL-3.0-or-later"
+license-files = ["LICENSE" ]
+authors = [
+    { name = "Nicolas Busca" },
+    { name = "Helion du Mas des Bourboux" },
+    { name = "Ignasi Pérez-Ràfols", email = "iprafols@gmail.com" },
+    { name = "Michael Walther" },
+    { name = "The DESI Lya forest picca topical group" }
+]
+keywords = ["cosmology", "lyman-alpha", "bao"]
+classifiers = [
+    "Programming Language :: Python :: 3"
+]
+
+[tool.setuptools.dynamic]
+version = {attr = "picca.__version__"}
+readme = {file = "README.md"}
+dependencies = {file = "requirements.txt"}
+
+[project.urls]
+Homepage = "https://github.com/igmhub/picca"
+
+[tool.setuptools]
+package-dir = { "" = "py" }
+
+[tool.setuptools.packages.find]
+where = ["py"]
+namespaces = true
+
+[tool.setuptools.package-data]
+picca = [
+    "fitter2/models/*/*.fits",
```

Comment:
I think we removed fitter2 at some point, so this line could go away

Reply:
It is still in setup.py, so this should be addressed separately to avoid obfuscation of changes